Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrType.kt

```diff
@@ -28,6 +28,20 @@ class ClassifierStubType(
     }
 }
 
+val ClassifierStubType.isTypealias: Boolean
+    get() = underlyingType != null
+
+val StubType.expandedType: StubType
+    get() {
+        if (this !is ClassifierStubType) return this
+        if (!isTypealias) return this
```

Comment:
Isn't it enough to use `var expandedType = this` below instead of these two checks?